201-front-door-standard-premium-app-service-public/main.bicep

```diff
@@ -0,0 +1,175 @@
+param location string = resourceGroup().location

```

Comment:
does bicep support metadata yet? portal picks that up when deploying templates...

Reply:
Yes it does support metadata. I have added descriptions to each parameter.